### DISCUSSION TITLE
Git gateway

### DISCUSSION THREAD

**Discussion by Matthijz98** [Thinkmill/keystatic#916] [Ideas]
I am looking for a cms system for my astro site.
Tryed Tina.io but don't like it for a couple reasons.

Want to to give Keystatic a go. But one thing i would like a lot is for my users to not use git at all and also do not have access to other content than inside the content collections folder.

Something like the gitgateway that decapcms has: https://decapcms.org/docs/git-gateway-backend/

**Comment by simonswiss** [comment]
Perhaps [Keystatic Cloud](https://keystatic.cloud) could be a good solution for you? It's free for up to 3 users.

**Reply by Matthijz98** [reply]
Where can i find more about the keystatic cloud?
Things like pricing and the user limit?

**Comment by Matthijz98** [comment]
I need 10+ users. But will give the cloud a go. But self hosted option would be great.
